### PR TITLE
Feature/task 370 target linux

### LINKED ISSUE #370: Create target platform definition for linux
Add a second target platform definition with linux specific operating- and windowing system.

### PR DESCRIPTION
**Pull request by Anna-Lena-DLR**
Added a tmodel and target for linux

changed working to windowing system in virsat.tmodel

closes #370

### CODE REVIEW

File: de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel [hunk 1]
```diff
@@ -1,6 +1,6 @@
- Target virsat {
+Target virsat {
```
Comment (franzTobiasDLR) [inline de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel:1]: What do you think of a virsat_windows target? To be consistent?

Comment (Anna-Lena-DLR) [reply de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel:1]: @SaMuellerDLR  what do you think about it?

Comment (SaMuellerDLR) [reply de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel:1]: @Anna-Lena-DLR Sure 👍

**Comment by SaMuellerDLR** [comment]
@Anna-Lena-DLR Is it right that we have 3 .target files? I can understand having 3 .tmodel files. But the third .target without an operating system set seems confusing to me.

**Comment by franzTobiasDLR** [comment]
Maybe we need an abstract tmodel... :D

**Comment by Anna-Lena-DLR** [comment]
So @SaMuellerDLR  we decided to change the virsat.tmodel to a targetLibrary which the windows and linux target are importing to get all the needed repositoryLocations. In downstream projects like CEF, TML etc. the user has to change the extending target to either the windows or the linux target, as they prefer. Is that fine for you?

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
There seem to be unnecessary changes e.g. in the javadoc api

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by franzTobiasDLR — APPROVED** [review]

**Pull request merged**